.circleci/config.yml

```diff
@@ -1,20 +0,0 @@
-version: 2
-jobs:
-  build:
-    docker:
-      - image: cimg/python:3.9
-    steps:
-      - checkout
-      - setup_remote_docker:
-          docker_layer_caching: false
-          version: 20.10.11
-      - run:
-          name: Setup
-          command: |
-            echo "export GIT_COMMIT=${CIRCLE_SHA1}" >> $BASH_ENV
```

Comment:
I think you might need to do something like this for the GHA so that the build has it

eg add a new step with

`echo "GIT_COMMIT=${CIRCLE_SHA1}" >> $GITHUB_OUTPUT`

Reply:
Yeah this may be needed to see the commit hash. I'll check it out. Thanks for the callout.

Reply:
Looks like this is working without this being explicitly set. See [set_git_env_vars.sh](https://github.com/mozmeao/basket/blob/master/docker/bin/set_git_env_vars.sh#L5)